lib/allocator/index.spec.ts

```diff
@@ -1,41 +1,39 @@
-import Web3 from 'web3'
-import { CustomOptions } from '../option'
+import { ethers } from 'ethers'
 import { createAllocatorContract, CreateAllocatorContract } from '.'
 import { createCalculateMaxRewardsPerBlockCaller } from './calculateMaxRewardsPerBlock'
 import { allocatorAbi } from './abi'
 
+jest.mock('./calculateMaxRewardsPerBlock')
+
 describe('allocator/index.ts', () => {
 	describe('createAllocatorContract', () => {
+		;(createCalculateMaxRewardsPerBlockCaller as jest.Mock).mockImplementation(
+			(contract) => contract
+		)
 		it('check return object', () => {
 			const host = 'localhost'
-			const client = new Web3()
-			client.setProvider(new Web3.providers.HttpProvider(host))
-
-			const expected: (
-				address?: string,
-				options?: CustomOptions
-			) => CreateAllocatorContract = (
-				address?: string,
-				options?: CustomOptions
+			const address = 'address'
+			const provider = new ethers.providers.JsonRpcProvider(host)
```

Comment:
@aggre 
Although it is test code, we want to make sure that there are no major changes in the behavior of the program as much as possible. So I used `ethers.providers.JsonRpcProvider` , is that OK?